src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -58,13 +52,7 @@ file that was distributed with this source code.
     <script>
```

Comment:
Would be great if we can completely remove the on page javascript an move this to the `admin.js` file. That would be necesarry if you use a strict CSP.

Reply:
Yes, I have added that to the issue #7049 

It needs a lot of work for some templates, we should probably take a look at previous PRs that tried it.